storage/src/main/java/tech/pegasys/teku/storage/server/network/DatabaseNetwork.java

```diff
@@ -39,12 +40,23 @@ public class DatabaseNetwork {
   @VisibleForTesting
   final String depositContract;
 
+  @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
i believe this goes on the class, then on the other 2 fields you can put `required = true` in their JsonProperty...

We could easily write a test case that shows the 2 first fields are required by opening an empty file or just putting in one of the fields and seeing the failure condition...